File: src/test/java/com/github/jactor/junit/GreetJUnitVintageTest.java
package com.github.jactor.junit;

import static org.assertj.core.api.AssertionsForClassTypes.assertThat;

import org.junit.Test;

public class GreetJUnitVintageTest {

    @Test
    public void  shouldGreetTheWorld() {
        assertThat(new Greet().sayHello()).isEqualTo("Hello World!");
    }

    @Test
    public void shouldGreetPerson() {
        assertThat(new Greet("Tor Egil").sayHello()).isEqualTo("Hello Tor Egil!");
    }
}

File: src/test/kotlin/com/github/jactor/junit/GreetJUnitVintageTest.kt
package com.github.jactor.junit

import org.assertj.core.api.Assertions.assertThat
import org.junit.Test

class GreetJUnitVintageTest {

    @Test
    fun `should greet the world`() {
        assertThat(Greet().sayHello()).isEqualTo("Hello World!")
    }

    @Test
    fun `should greet a person`() {
        assertThat(Greet("Tor Egil").sayHello()).isEqualTo("Hello Tor Egil!")
    }
}